cfpb/consumerfinance.gov
# PfC: Open link in new tab

Updates a PfC financial path link to open in new tab to avoid interrupting user interaction with tool.


## Changes

- Adds target and rel attributes to link


## How to test this PR

1. Check out branch
2. After filling out first screen, navigate to "Other sources" tab under "Your financial aid offer". Verify that "learn more" link opens in new tab.

## Screenshots
<img width="1026" alt="Screen Shot 2021-01-19 at 11 25 44 AM" src="https://user-images.githubusercontent.com/778171/105082931-1b2d5400-5a49-11eb-98c0-21fefc8c8029.png">
